st2api/st2api/controllers/v1/action_views.py

```diff
@@ -88,7 +88,6 @@ def _get_one(action_id, requester_user):
                 GET /actions/views/parameters/1
         """
         action_db = LookupUtils._get_action_by_id(action_id)
-        LOG.info('Found action: %s, runner: %s', action_db, action_db.runner_type['name'])
```

Comment:
This reduces duration of this request for 10 seconds where there are many actions in the database.